setup.py

```diff
@@ -86,6 +84,10 @@
             "sphinx-rtd-theme",
             "sphinxcontrib-apidoc",
         ],
+        "render": [
```

Comment:
I am unsure what the name should be: `render`? `renderer`? `camera`? `camera-obs`? `rendering`?

Reply:
Maybe, this is something we can ask others,